### PR TITLE
New job operator start method takes properties as a parameter

### PR DESCRIPTION
**Pull request by CNJingo**
This PR is related to #4304

### CODE REVIEW

**Comment by fmbenhassine** [comment]
Hi @CNJingo ,

Thank you for your PR! I think we were working on this at the same time :-) This was resolved in [a547fe5](https://github.com/spring-projects/spring-batch/commit/a547fe5b0a8c72ffbb6349485478a5c104db99e9).

I appreciate your effort here! Could you please update the PR to keep only the changes that remove the usage of the deprecated method? Please rebase it on the latest `main`. After that, it will be good to merge.

Many thanks upfront!

**Comment by CNJingo** [comment]
Okay I will update the PR soon.

**Comment by CNJingo** [comment]
I resolve it.

File: spring-batch-core/src/test/java/org/springframework/batch/core/launch/support/SimpleJobOperatorTests.java [hunk 1]
```diff
@@ -162,18 +168,23 @@ void testStartNextInstanceSunnyDay() throws Exception {
 
 	@Test
 	void testStartNewInstanceSunnyDay() throws Exception {
-		jobParameters = new JobParameters();
+		Properties parameters = new Properties();
+		parameters.setProperty("a", "a=b");
```
Comment (fmbenhassine) [inline spring-batch-core/src/test/java/org/springframework/batch/core/launch/support/SimpleJobOperatorTests.java:172]: This should be `parameters.setProperty("a", "b");`. I will amend that when merging the PR.

File: spring-batch-samples/src/test/java/org/springframework/batch/sample/launch/RemoteLauncherTests.java [hunk 2]
```diff
@@ -17,6 +17,7 @@
 
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
+import org.hibernate.query.sql.spi.ParameterOccurrence;
```
Comment (fmbenhassine) [inline spring-batch-samples/src/test/java/org/springframework/batch/sample/launch/RemoteLauncherTests.java:20]: This import is not needed, I will remove it on merge.

**Comment by fmbenhassine** [comment]
Rebased and merged as 2cc0be4a0c31b7bc190e7244145552cc3a0927b0. Thank you for your contribution!

File: spring-batch-core/src/test/java/org/springframework/batch/core/launch/support/SimpleJobOperatorTests.java (hunk 1 above)
Comment (CNJingo) [reply spring-batch-core/src/test/java/org/springframework/batch/core/launch/support/SimpleJobOperatorTests.java:172]: Thank you for correcting my mistake.

**Pull request closed without merging**